tests/Paramore.Brighter.RMQ.Tests/MessagingGateway/When_confirming_posting_a_message_via_the_messaging_gateway.cs

```diff
@@ -74,7 +73,7 @@ public RmqMessageProducerConfrmationsSendMessageTests ()
         [Fact]
         public async Task When_confirming_posting_a_message_via_the_messaging_gateway()
         {
-            _messageProducer.Send(_message);
+            await _messageProducer.SendAsync(_message);
```

Comment:
We don't support SendAsync on an RMQ producer as there is no Async method on RMQ. I'll revert then patch a fix to use a taskcompletion source so at least that code path won't throw for now